### PR TITLE
correct DO_CHANGE_SPEED descriptions

### PR DESCRIPTION
**Pull request by Hwurzburg**
@rmackay9 in addition to reviewing this, why do we have https://ardupilot.org/copter/docs/mission-command-list.html and this page? I see you have mission planner shots which are nice but outdated and some incorrect now, but this is duplication and a maintenance issue...

I think it might be better to expand the copter section of the common page with any missing information (if any) for commands, put a couple of mission planner shots showing a few waypoint type shots for commands at the top of the common page to give someone the info on where/how to change in mission planner and eliminate that page....lets discuss when you get time soon...

### CODE REVIEW

File: common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst [hunk 1]
```diff
@@ -2848,7 +2848,7 @@ set to 0).
    <tr style="color: #c0c0c0">
    <td><strong>param1</strong></td>
    <td>Type (0=as 1=gs)</td>
-   <td>Speed type (0=Airspeed, 1=Ground Speed). Set to 0 before v2.50, otherwise ignored.</td>
+   <td>Speed type . Set to 0 before v2.50, otherwise ignored.</td>
```
Comment (rmackay9) [inline common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst:2851]: we probably don't need to mention v2.50 anymore.

**Comment by rmackay9** [comment]
@Hwurzburg I'm happy to have just a single page for all vehicles but I find the copter page is still better than the combined page.  

In particular the Copter page gets to the commands very high up on the page while the combined page requires the user to scroll down through many pages of detailed discussion before getting to the commands.

File: common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst]: fixed

**Comment by Hwurzburg** [comment]
merging....as to single page...will work on that once @Gone4Dirt finished reviewing our mavlink command docs for accuracy...I will then update, then reorg in a manner similar to the standalone copter page for better navigation....finally archiving that page

**Pull request merged**